### PR TITLE
support render chart at Zeppelin and nteract

### PR DESCRIPTION
**Pull request by sunhailin-Leo**
* Support render at Zeppelin and nteract. https://github.com/pyecharts/pyecharts/issues/726

### CODE REVIEW

File: test/test_bar.py [hunk 1]
```diff
@@ -157,3 +161,27 @@ def test_bar_graphic(fake_writer):
...
+def test_bar_render_zeppelin():
+    CurrentConfig.NOTEBOOK_TYPE = NotebookType.ZEPPELIN
+    c = (
+        Bar()
+        .add_xaxis(["A", "B", "C"])
+        .add_yaxis("series0", [1, 2, 4])
+        .add_yaxis("series1", [2, 3, 6])
+    )
+    zeppelin_code = c.render_notebook()
+    eq_(zeppelin_code, None)
```
Comment (chfw) [inline test/test_bar.py]: This assertion should intercept console output and check if %html is present. It is a bit difficult but not impossible. Google online or checkout my monban project where there exist some examples.

Comment (sunhailin-Leo) [reply test/test_bar.py]: I know How to block it, not difficult.

* First: Create a redirector
```python
import sys


class ConsoleOutputRedirect:
    """ Wrapper to redirect stdout or stderr """

    def __init__(self, fp):
        self.fp = fp

    def write(self, s):
        self.fp.write(s)

    def writelines(self, lines):
        self.fp.writelines(lines)

    def flush(self):
        self.fp.flush()


stdout_redirect = ConsoleOutputRedirect(sys.stdout)
```

* Second: Use StringIO to keep the sys.stdout
```python
def test_bar_render_zeppelin():
    CurrentConfig.NOTEBOOK_TYPE = NotebookType.ZEPPELIN
    c = (
        Bar()
        .add_xaxis(["A", "B", "C"])
        .add_yaxis("series0", [1, 2, 4])
        .add_yaxis("series1", [2, 3, 6])
    )
    # Block Console stdout
    output_buffer = StringIO()
    stdout_redirect.fp = output_buffer
    temp_stdout = sys.stdout
    sys.stdout = stdout_redirect

    # render
    c.render_notebook()
    sys.stdout = temp_stdout

    # Block Result
    assert_in("%html", output_buffer.getvalue())
```

Comment (chfw) [reply test/test_bar.py]: Let me share some examples offline. 

I am not trying to block your PR. Just want to point out that the test thing could be done better. At the end of the day, it is the coding experience benefiting us all. We are not paid through the work.

File: test/test_bar.py [hunk 2]
```diff
@@ -157,3 +164,35 @@ def test_bar_graphic(fake_writer):
...
+def test_bar_render_zeppelin():
+    CurrentConfig.NOTEBOOK_TYPE = NotebookType.ZEPPELIN
+    c = (
+        Bar()
+        .add_xaxis(["A", "B", "C"])
+        .add_yaxis("series0", [1, 2, 4])
+        .add_yaxis("series1", [2, 3, 6])
+    )
+    # Block Console stdout
+    stdout_redirect.fp = StringIO()
```
Comment (chfw) [inline test/test_bar.py]: Here's the alternative to use mock but you got the idea now.

```

def test_partial_run():
--
16 | patcher = patch("sys.stdout", new_callable=StringIO)
17 | fake_stdout = patcher.start()
18 | reporter.report_partial_run(1, 20)
19 | patcher.stop()
20 | eq_(fake_stdout.getvalue(), "Templated 1 out of 20 files.\n")


```

**Pull request merged**